src/ansys/fluent/core/services/field_data.py

```diff
@@ -1053,8 +1258,10 @@ def get_vector_field_data(
 
         Returns
         -------
-        Dict[int, Tuple[np.array, float]]
-            Dictionary containing a map of surface IDs to a tuple of vector field and vector scale.
+        Union[VectorFieldData, Dict[int, VectorFieldData]]
+            If a surface name is provided as input, vector field data.
+            If surface IDs are provided as input, a dictionary containing a map of
+            surface IDs to vector field data.
```

Comment:
```suggestion
        Union[VectorFieldData, Dict[int, VectorFieldData]]
            If a surface name is provided as input, vector field data is returned.
            If surface IDs are provided as input, a dictionary containing a map of
            surface IDs to vector field data is returned.
```
Again, I think adding "is returned" to the two options helps with understanding.